dlk/tests/test_code_generation.py

```diff
@@ -416,7 +417,7 @@ def codegen_cpu(self,
 class TestCodeGenerationX8664(TestCodeGenerationBase):
     """Test class for code generation testing."""
 
-    @params(*get_configurations_x86_64())
+    @parameterized.expand(get_configurations_x86_64())
```

Comment:
Can't use `@pytest.mark.parametrize`?

Reply:
I've tried it first, but unfortunately, `@pytest.mark.parametrize` does not work in `unittest.TestCase` subclasses.
https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses
`import parameterized` is an easy way to remove nose2 dependency without large structural changes(e.g. removing inheritance of unittest.TestCase).

Reply:
OH... Thank you for your explanation!!